server/views/taskList/caSubmitTask.pug

```diff
@@ -1,23 +1,23 @@
+- var canSendToDm = licenceStatus.stage === 'PROCESSING_CA' || allowedTransitions.caToDm
+- var canSendToRo = licenceStatus.stage === 'ELIGIBILITY' && allowedTransitions.caToRo
+
```

Comment:
this isn't needed - allowed transitions tells you what's allowed

Maybe move the stage check into the allowedtransitions logic to make it a tighter spec, but the view should just use allowedtransitions